### DISCUSSION TITLE
I want to achieve this effect （button group）

### DISCUSSION THREAD

**Discussion by junmaqiang** [fyne-io/fyne#4276] [Q&A]
# Is there any reference plan?
![WX20230923-171134](https://github.com/fyne-io/fyne/assets/1819457/045dcef0-6005-468b-ad5d-4202365b3d54)

**Comment by Jacalz** [comment]
It is not something that we have implemented yet, no. Either open a feature request or create your own custom widget. It should definitely be doable, especially now with rounded rectangles being supported.

**Comment by dweymouth** [comment]
I did something like this as a custom widget for Supersonic implemented by packing builtin Button widgets together with no padding and using the Importance property to highlight one: https://github.com/dweymouth/supersonic/blob/main/ui/widgets/togglebuttongroup.go

It won't work well with rounded rectangles so I plan on trying to make a proper SegmentedButton widget as part of migrating Supersonic to Fyne 2.4.x (which hasn't been done yet)

**Reply by junmaqiang** [reply]
your project is good , very nice ,so cool
